### PR TITLE
Remove webjobs package reference

### PR DESCRIPTION
**Pull request by pragnagopa**

### CODE REVIEW

**Comment by pragnagopa** [comment]
cc @gohar94 - this should unblock PR build. Thanks to @brettsam

File: src/Microsoft.Azure.WebJobs.Extensions.Storage/WebJobs.Extensions.Storage.csproj [hunk 1]
```diff
@@ -96,9 +96,13 @@
...
     <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="11.1.7" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.14" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta004">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Azure.WebJobs.Host\WebJobs.Host.csproj" />
+    <ProjectReference Include="..\Microsoft.Azure.WebJobs\WebJobs.csproj" />
```
Comment (brettsam) [inline src/Microsoft.Azure.WebJobs.Extensions.Storage/WebJobs.Extensions.Storage.csproj]: You should be able to remove `WebJobs.csproj` -- it'll be pulled in automatically.

**Review by brettsam — APPROVED** [review]
Just one minor comment.

**Review by brettsam — APPROVED** [review]

**Comment by pragnagopa** [comment]
cc @mhoeger

**Pull request merged**